sql_warning.txt

```diff
@@ -0,0 +1,60 @@
+Unable to parse INSERT INTO "scenario_data" ("data") VALUES (?) ON CONFLICT ("sha_256") DO UPDATE SET "sha_256" = DEFAULT RETURNING "sha_256" : Syntax error found near Numbered Bind Parameter (VALUES Clause)
+Unable to parse INSERT INTO "scenario_data" ("data") VALUES (?::json) ON ... : Syntax error found near Custom Datatype Name (VALUES Clause)
+Unable to parse UPDATE "scenarios" SET "data" = '{"version":"1.2","classMap":[{"name":"controllers","location":"app/controllers","type":"package","children":[{"name":"SessionController","location":"app/controllers/session_controller.rb:2","type":"class","children":[{"name":"create","location":"app/controllers/session_controller.rb:4","type":"function","static":false}]}]},{"name":"models","location":"app/models","type":"package","children":[{"name":"Session","location":"app/models/session.rb:2","type":"class","children":[{"name":"initialize","location":"app/models/session.rb:4","type":"function","static":false},{"name":"delete","location":"app/models/session.rb:8","type":"function","static":true}]}]}],"metadata":{"name":"Example scenario","language":{"name":"ruby"},"recorder":{"name":"rspec"},"client":{"name":"appmap","url":"https://github.com/applandinc/appmap-ruby","inferred":true}},"events":[{"id":1,"event":"call","defined_class":"SessionController","method_id":"create","path":"app/controllers/session_controller.rb","lineno":4,"receiver":{"class":"SessionController","value":"#<SessionController:0x0000000000000000>","object_id":"0"},"http_server_request":{"request_method":"POST","normalized_path_info":"/sessions/new","path_info":"/sessions/new","client_normalized_path_info":"/sessions/new"},"message":[{"name":"login","class":"String","object_id":70252828231460,"value":"alice"}]},{"id":2,"event":"call","defined_class":"Session","method_id":"initialize","path":"app/models/session.rb","lineno":4,"receiver":{"class":"Session","value":"#<Session:0x0000000000000000>","object_id":"1"},"sql_query":{"sql":"SELECT * FROM users WHERE login = ''alice''","normalized_sql":"SELECT * FROM users WHERE login = $1","normalized":true}},{"id":3,"event":"return","defined_class":"Session","method_id":"initialize","path":"app/models/session.rb","lineno":4,"parent_id":2},{"id":4,"event":"return","defined_class":"SessionController","method_id":"create","path":"app/controllers/session_controller.rb","lineno":4,"parent_id":1}]}'::jsonb, "sha_256" = '\x... : Syntax error found near Column Identifier (SET Clause)
```

Comment:
Looks like something happened with this commit. Should these files be added? I don't see any changes to the finding hash.

Reply:
Yeah, this happened during rebase for some reason. I've dropped the whole commit, and the 7665c36 commit as well.